<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cyclock</class>
 <widget class="QMainWindow" name="cyclock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cyclock</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(97, 53, 131);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0,1,0,1">
    <property name="spacing">
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="font">
         <font>
          <family>Ubuntu</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 240, 107);
color: rgb(26, 95, 180);</string>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>132</pointsize>
         </font>
        </property>
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 240, 107);
color: rgb(26, 95, 180);</string>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>132</pointsize>
         </font>
        </property>
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdNumber_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 240, 107);
color: rgb(26, 95, 180);</string>
        </property>
        <property name="digitCount">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
